apps/perf-test-react-components/src/scenarios/ListItem.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
```

Comment:
no new code should be added to these scenarios as they bring no value atm. @spmonahan tensile tool will be used in the future. you can close this PR

Reply:
ok, thank you! I'll update the template for the component release process to remove this information